README.md

```diff
@@ -720,9 +720,11 @@ public class Variant
 }
 ```
 
-### Getting a Feature's Variant
+### Getting Variants
 
-A feature's variant can be retrieved using the `IVariantFeatureManager`'s `GetVariantAsync` method.
+For each feature, a variant can be retrieved using the `IVariantFeatureManager`'s `GetVariantAsync` method. The variant returned is dependent on the
+user currently being evaluated, and that information is obtained from an instance of `TargetingContext`. This context can either be passed in when
+calling `GetVariantAsync` or it can be automatically retrieved from an implementation of `ITargetingContextAccessor` if one is registered.

```

Comment:
I feel that this should link to the `ITargetingContextAccessor` section for more details.